functional/security/ssl-tests/build.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0"?>
+
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+
+<project name="ssl-tests Security Functional tests" default="build" basedir=".">
+    <taskdef resource="net/sf/antcontrib/antlib.xml" />
+    <description>
+        ssl-tests Security Functional tests
+    </description>
+
+    <!-- set global properties for this build -->
+    <property name="DEST" value="${BUILD_ROOT}/functional/security/ssl-tests" />
+
+    <!--Properties for this particular build-->
+    <property name="src" location="./ssl-tests" />
+
+    <target name="ssltests.check">
+        <condition property="ssltestsTestdir.exists">
+            <available file="ssl-tests" type="dir"/>
+        </condition>
+    </target>
+
+    <if>
+        <contains string="${SPEC}" substring="zos"/>
+        <then>
+            <property name="GIT_REPO" value="git@github.com:" />
+        </then>
+        <else>
+            <property name="GIT_REPO" value="https://github.com/" />
+        </else>
+    </if>
+    <target name="getSsltestsTest" depends="ssltests.check" unless="ssltestsTestdir.exists">
+        <exec executable="git" failonerror="true">
+            <arg value="clone" />
+            <arg value="${GIT_REPO}rh-openjdk/ssl-tests.git" />
+        </exec>
+    </target>
+
+    <target name="init">
+        <mkdir dir="${DEST}" />
+    </target>
+    <if>
+        <or>
+            <matches pattern="^[89]{1}$" string="${JDK_VERSION}"/>
+            <matches pattern="^[1][023456]" string="${JDK_VERSION}"/>
+        </or>
+        <then>
+            <property name="jtregTar" value="jtreg_5_1_b01"/>
+        </then>
+        <else>
+            <if>
+                <matches pattern="^[1][17]" string="${JDK_VERSION}"/>
+                <then>
+                    <property name="jtregTar" value="jtreg_6_1"/>
+                </then>
+                <else>
+                    <if>
+                        <matches pattern="^[1][89]" string="${JDK_VERSION}"/>
+                        <then>
+                            <property name="jtregTar" value="jtreg_6_1_1"/>
+                        </then>
+                        <else>
+                            <property name="jtregTar" value="jtreg_7_1_1_1"/>
```

Comment:
jtreg_7_3_1_1 should be used for JDK21+.

Please check https://github.com/adoptium/aqa-tests/blob/master/openjdk/build.xml#L44-L67 for jtreg version.